source/plugins/ruby/in_win_cadvisor_perf.rb

```diff
@@ -8,6 +8,7 @@ class Win_CAdvisor_Perf_Input < Input
     Fluent::Plugin.register_input("win_cadvisor_perf", self)
 
     @@winNodes = []
+    @@totalPerfCount = 0
```

Comment:
instance variable would be good enough for totalPerfCount since this being shared other instances of the plugins, right?